lib/controllers/ServoMotor.cpp

```diff
@@ -1,20 +0,0 @@
-#include "ServoMotor.h"
```

Comment:
@jetkov Hey Alex, do you mind if I get rid of the ServoMotor class since servo is implemented under actuator-controller?

Reply:
I'd much prefer if you use the ServoMotor class instead of modifying the Motor class. The point of ActuatorController was to allow generic sensors (input) and actuators (output) to interface. You really shouldn't make one class try to be two.

Reply:
The ClawController app also depends on the ServoMotor class, which is why I wrote it in the first place.